### PR TITLE
Fan Club Membership SDK functionality

### PR DESCRIPTION
**Pull request by rado196**

### CODE REVIEW

File: apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts [hunk 1]
```diff
@@ -303,9 +303,9 @@ class FanClubMembershipController extends BaseController {
     fanClubId,
     memberId,
   }: FanClubMembershipRequestParams): FanClubMembershipRequestResult {
-    let isBulkAction = false;
+    let isBulkAction = true;
     if (!Array.isArray(fanClubId)) {
-      isBulkAction = true;
+      isBulkAction = false;
```
Comment (rubenaprikyan) [inline apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts:308]: why we need bulkAction here?

File: sdks/core-api-sdk/src/sdks/FanClubMembershipSDK/index.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  // #endregion
+
+  // #region member actions
+  public acceptInvitation(fanClubId: DbIdentifier, membershipId: DbIdentifier) {
+    return this.api?.makeAPIPatchRequest<ConfirmationResultType>(
+      `${fanClubId}/memberships/${membershipId}/accept-invitation`
+    );
+  }
+
+  public rejectInvitation(fanClubId: DbIdentifier, membershipId: DbIdentifier) {
```
Comment (rubenaprikyan) [inline sdks/core-api-sdk/src/sdks/FanClubMembershipSDK/index.ts:131]: could you please write function description? and js doc for params

**Review by rubenaprikyan — APPROVED** [review]
looks good!

File: apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts (hunk 1 above)
Comment (rado196) [reply apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts:308]: I just write dynamic create action, if provided data is single then will be created only one row, but if the client-side send us a list of rows in the same route then we can execute the action for bulk insertion.

Just have one route for single and bulk action.

File: sdks/core-api-sdk/src/sdks/FanClubMembershipSDK/index.ts (hunk 2 above)
Comment (rado196) [reply sdks/core-api-sdk/src/sdks/FanClubMembershipSDK/index.ts:131]: sure.

File: apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts (hunk 1 above)
Comment (rubenaprikyan) [reply apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts:308]: So what's the use-case of using bulk action?

Comment (rado196) [reply apis/core-api/src/core/controllers/FanClubMembershipController/FanClubMembershipController.ts:308]: In UI we don't have bulk actions for now, but maybe someday we need functionality to allow members to select more than one fan club and request to join.

**Review by rubenaprikyan — APPROVED** [review]
Lgtm

**Pull request merged**